tests/robustness/traffic/etcd.go

```diff
@@ -155,6 +156,35 @@ func (t etcdTraffic) Run(ctx context.Context, c *client.RecordingClient, limiter
 	}
 }
 
+func (t etcdTraffic) RunCompactLoop(ctx context.Context, c *client.RecordingClient, period time.Duration, finish <-chan struct{}) {
+	var lastRev int64 = 2
+	timer := time.NewTimer(period)
```

Comment:
```suggestion
	timer := time.NewTimer(period)
	defer timer.Stop()
```

Reply:
Or use time.After like in Kubernetes

Reply:
I think `time.Ticker` is more suitable here.

At first I also didn't want to change this code as it is just move from `tests/robustness/traffic/traffic.go` to here, but it is as good time as any to fix it.